packages/block-library/src/post-comment-edit/index.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comment-edit` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comment-edit` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Return the post comment's date.
+ */
+function render_block_core_post_comment_edit( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['commentId'] ) || ! current_user_can( 'edit_comment', $block->context['commentId'] ) ) {
+		return '';
+	}
+
+
+	$edit_comment_link = get_edit_comment_link( $block->context['commentId'] );
+	$open_in_new_tab   = $attributes['openInNewTab'] === true;
+
+	$link_atts = '';
+
+	if ( $open_in_new_tab ) {
+		$link_atts .= 'target="_blank"';
+	}
+
+	return sprintf(
+		'<div %1$s><a href=%2$s %3$s>%4$s</a></div>',
```

Comment:
The link href value should have quotes around it, as otherwise it produces this HTML:
<img width="691" alt="Screenshot 2021-04-12 at 12 18 03 pm" src="https://user-images.githubusercontent.com/677833/114340040-4081b800-9b89-11eb-9a9f-41a73662e24e.png">